site/docs/foundations/iconography/country-symbols.mdx

```diff
@@ -0,0 +1,36 @@
+---
+title: Country symbols
+layout: DetailTechnical
+---
+
+Country symbols represent a specific country and are used in contexts where geographical identification is necessary. They can be considered icons that are visually representative of a country or inter-governmental organisation. All country symbols are available in both circle and rectangle shapes.
+
+<Image
+  src="/img/foundations/iconography-countrysymbols-variant.png"
+  alt="Example of circular country symbol next to rectangular country symbol"
+/>
+
+## All country symbols
+
+<CountrySymbolPreview />
+
+## Scaling country symbols
+
+Country symbols have a standard size that changes in multiples across all four density levels.
+
+<ImageSwitcher
+  images={[
+    { src: "/img/foundations/scale-country.svg", alt: "" },
+    { src: "/img/foundations/scale-country-annotated.svg", alt: "" },
+  ]}
+  label="Show multipliers"
+/>
+
+| Size token         | Multiplier | High density (px) | Medium | Low | Touch |
```

Comment:
Is it an issue that High is the only one with the suffix and unit?

Reply:
Edited it to match other table in foundations